controllers/base_controller.go

```diff
@@ -225,8 +281,10 @@ func (r *BaseReconciler) pollSpire(out chan event.GenericEvent, s <-chan struct{
 								log.Error(err, "Unable to fetch resource", "name", namespacedName)
```

Comment:
Any implication here? Need to monitor / alert on this happening?

Reply:
It's the kind of error you go looking for when it just doesn't work, it shouldn't happen unless the k8s api is throwing errors.